tools/test/audio/process_test.m

```diff
@@ -409,6 +409,9 @@
 	test.thdnf_max = [];            % Set per component
```

Comment:
Minor nit: the "Fixes commit-id" tag not quite following Linux style. 

```
If your patch fixes a bug in a specific commit, e.g. you found an issue using
``git bisect``, please use the 'Fixes:' tag with the first 12 characters of
the SHA-1 ID, and the one line summary.  Do not split the tag across multiple
lines, tags are exempt from the "wrap at 75 columns" rule in order to simplify
parsing scripts.  For example::

»       Fixes: 54a4f0239f2e ("KVM: MMU: make kvm_mmu_zap_page() return the number of pages it actually freed")                                          
```

Reply:
There are git scripts that scan branches for that specific `Fixes:` syntax.

Reply:
The commit message is now fixed. You may want to re-review because of 2nd commit added to fix the syntax error found by Sriram. And strange that it didn't error.